### DISCUSSION TITLE
Project unattention

### DISCUSSION THREAD

**Discussion by eduardofilo** [eduardofilo/RG350_adam_image#314] [General]
A small statement: As you can see, the image has not received any attention in recent months. The team behind Adam is busy with other projects. But the image is not abandoned. When the situation allows it, the idea is to publish a new release with the latest OBbeta builds (which has several improvements, including a performance increase), a version with stability improvements of SimpleMenu and the usual RetroArch update. Also issues and discussions opened during the last weeks that have not been answered/resolved will be taken care of. Sorry for the delay.

Una pequeña declaración: Como se puede observar, la imagen no ha recibido atención los últimos meses. El equipo que hay detrás de Adán está ocupado con otros proyectos. Pero la imagen no está abandonada. Cuando la situación lo permita, la idea es lanzar una nueva release con las últimas compilaciones de OBbeta (que tiene varias mejoras, entre ellas una mejora de rendimiento), una versión con mejoras de estabilidad de SimpleMenu y la habitual actualización de RetroArch. También se atenderán las incidencias y discusiones abiertas durante las últimas semanas que no han obtenido respuesta/solución. Disculpad el retraso.

**Comment by zugurudumba** [comment]
You guys are awesome, thank you for all the work put into this image. What other interesting projects are you guys working on?

**Reply by eduardofilo** [reply]
A distribution (not image because its architecture) for Miyoo Mini.

**Reply by zugurudumba** [reply]
That sounds awesome, can't wait to try it on my Miyoo Mini!
